specification/web/resource-manager/Microsoft.CertificateRegistration/stable/2021-02-01/AppServiceCertificateOrders.json

```diff
@@ -0,0 +1,1727 @@
+          "readOnly": true
+        },
+        "notBefore": {
+          "format": "date-time",
+          "description": "Date Certificate is valid from.",
+          "type": "string",
+          "readOnly": true
+        },
+        "notAfter": {
+          "format": "date-time",
+          "description": "Date Certificate is valid to.",
+          "type": "string",
+          "readOnly": true
+        },
+        "signatureAlgorithm": {
+          "description": "Certificate Signature algorithm.",
+          "type": "string",
+          "readOnly": true
+        },
+        "issuer": {
+          "description": "Certificate Issuer.",
+          "type": "string",
+          "readOnly": true
+        },
+        "rawData": {
+          "description": "Raw certificate data.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "CertificateEmail": {
+      "description": "SSL certificate email.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../Microsoft.Web/stable/2021-02-01/CommonDefinitions.json#/definitions/ProxyOnlyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "CertificateEmail resource specific properties",
+          "type": "object",
+          "properties": {
+            "emailId": {
+              "description": "Email id.",
+              "type": "string"
+            },
+            "timeStamp": {
+              "format": "date-time",
+              "description": "Time stamp.",
+              "type": "string"
+            }
+          },
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "CertificateOrderAction": {
+      "description": "Certificate order action.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../Microsoft.Web/stable/2021-02-01/CommonDefinitions.json#/definitions/ProxyOnlyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "CertificateOrderAction resource specific properties",
+          "type": "object",
+          "properties": {
+            "actionType": {
+              "description": "Action type.",
+              "enum": [
+                "CertificateIssued",
+                "CertificateOrderCanceled",
+                "CertificateOrderCreated",
+                "CertificateRevoked",
+                "DomainValidationComplete",
+                "FraudDetected",
+                "OrgNameChange",
+                "OrgValidationComplete",
+                "SanDrop",
+                "FraudCleared",
+                "CertificateExpired",
+                "CertificateExpirationWarning",
+                "FraudDocumentationRequired",
+                "Unknown"
+              ],
+              "type": "string",
+              "readOnly": true,
+              "x-ms-enum": {
+                "name": "CertificateOrderActionType",
+                "modelAsString": false
```

Comment:
>false [](http://example.com/codeflow?start=33&length=5)

Recommended value for modelAsString is true. I see this is consistently false for all enums in this PR, so presumably you have a reason for that preference.

Reply:
This is not a change from previous versions. If we change modelAsString to true now is that going to cause more issues for customers with backwards compatibility?